Microsoft.Toolkit.Uwp.SampleApp/Shell.SamplePicker.cs

```diff
@@ -36,11 +35,11 @@ private Sample CurrentSample
             set
             {
                 _currentSample = value;
-                var nop = SetNavViewSelection();
+                SetNavViewSelection();
             }
         }
 
-        private async Task SetNavViewSelection()
+        private async void SetNavViewSelection()
```

Comment:
We shouldn't have async void methods. If the method that we're awaiting throws an exception the entire application crashes. Compare that to `_ = SetNavViewSelection();`. If the method throws an exception, it's business as usual.

Reply:
Inside `Samples.GetCategoryBySample` add the following to the first line `throw new InvalidOperationException();`. Run the application and what happens when you select an item?

Reply:
We are using a discard to *not* use a pokemon catch in the fire-and-forget method. Not sure about what the best approach is in term of generated code but in the end, we have the same behavior.
Still, I've reverted to the old style.